DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/Index.cshtml

```diff
@@ -216,7 +216,7 @@
           Training venues
         </dt>
         <dd class="nhsuk-summary-list__value">
-          <partial name="_CentreStringArray" model="Model.TrainingVenues"/>
+          <span class="preserve-whitespace">@Model.TrainingVenues</span>

```

Comment:
Is the span needed/could we add the preserve-whitespace class on the dd element?

Reply:
I tried that first. The autoformatting will indent the value in the content on the dd on a new line, the indent will add extra space that will then be preserved in the HTML element, so that span does seem necessary.